core/src/test/java/com/exadel/aem/toolkit/bundle/optionprovider/services/impl/OptionProviderTest.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exadel.aem.toolkit.bundle.optionprovider.services.impl;
+
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.sling.api.resource.Resource;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.exadel.aem.toolkit.bundle.optionprovider.services.OptionProviderService;
+
+import io.wcm.testing.mock.aem.junit.AemContext;
+
+public class OptionProviderTest {
+
+    private static final String PROPERTY_VALUE = "value";
```

Comment:
isn't the constant from 'CoreConstants' reachable from here?

Reply:
Yes, I've missed it